### PR TITLE
UHM-7475, record the patient's mother using the new relationships widget

### PR DESCRIPTION
**Pull request by cioan**

### CODE REVIEW

File: configuration/pih/pih-config-sierraLeone.json [hunk 1]
```diff
@@ -45,6 +45,16 @@
         "required": false
       }
     },
+    "relationships" : [
+      {
+        "type": "mother",
```
Comment (mseaton) [inline configuration/pih/pih-config-sierraLeone.json:50]: Can you describe what this "type" property is used for?

Comment (mogoodrich) [reply configuration/pih/pih-config-sierraLeone.json:50]: See my comments on the Registration App PR, but, yeah, same question... I was assuming this was going to be used to determine whether we were selecting either the "a" (ie Parent) or "b" (ie Child) in the relationship.

Comment (cioan) [reply configuration/pih/pih-config-sierraLeone.json:50]: When setting up the question and the field on the form it needs a few identification attributes:

q.setId("relationships_" + relationship.getType());
https://github.com/PIH/openmrs-module-pihcore/pull/454/files#diff-92cd416097d1562de890998a976d0800d9921bdaa3ab8c3d2004f7ea32b568feR57

field.setFormFieldName(relationship.getType() + "Name");
https://github.com/PIH/openmrs-module-pihcore/pull/454/files#diff-92cd416097d1562de890998a976d0800d9921bdaa3ab8c3d2004f7ea32b568feR62

I could use relationTypeUuid, but that would be harder to read/understand. I thought that by using a simple common word would be easier to read the configuration file to determine what kind of relationships we are trying to configure. I guess we could use label too, but that is usually a little longer word.

Comment (cioan) [reply configuration/pih/pih-config-sierraLeone.json]: That makes sense, @mseaton ! I have renamed type to id. Thanks!

**Pull request merged**